server/app/models/userModel.js

```diff
@@ -0,0 +1,51 @@
+import mongoose, { Schema } from 'mongoose';
+
+import bcrypt from 'bcryptjs';
+
+const UserSchema = new Schema(
+  {
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    email: { type: String, required: true },
+    password: { type: String, required: true },
+  },
+  {
+    JSON: {
+      virtuals: true,
+    },
+  }
+);
+
+UserSchema.method('toJSON', function () {
+  const obj = this.toObject();
+  delete obj.password;
```

Comment:
Is the idea here you're just deleting the $819y58291y5981y51892y5981y5 garbage whenever we ask for a json version of the obj?

Reply:
Yeah exactly